pkg/kubectl/rollout_status.go

```diff
@@ -36,32 +34,26 @@ type StatusViewer interface {
 }
 
 // StatusViewerFor returns a StatusViewer for the resource specified by kind.
-func StatusViewerFor(kind schema.GroupKind, c kubernetes.Interface) (StatusViewer, error) {
+func StatusViewerFor(kind schema.GroupKind) (StatusViewer, error) {
 	switch kind {
 	case extensionsv1beta1.SchemeGroupVersion.WithKind("Deployment").GroupKind(), apps.Kind("Deployment"):
```

Comment:
While you're at it, please switch that `apps` to use `appsv1` here and in the next ones. That will remove yet another link with k8s internals.

Reply:
Fixed that in a separate PR: https://github.com/kubernetes/kubernetes/pull/68641